Gemfile.jruby-2.6.lock.release

```diff
@@ -903,8 +896,15 @@ DEPENDENCIES
   logstash-integration-jdbc
   logstash-integration-kafka
   logstash-integration-rabbitmq
+  logstash-mixin-ca_trusted_fingerprint_support
```

Comment:
These lines are interesting to me. AFAIK mixin is already included and now sure why they are not included in prev versions?

Reply:
Neither of this mixin were listed, I think was due to the old requirement:
```
logstash-output-elasticsearch (>= 11.6.0)
```